cli.js

```diff
@@ -60,7 +60,7 @@ const generateConfig = (options) => {
 
   getStackDetails(samConfig, samConfigEnv)
     .then(details => getOutputs(details.stackName, details.region))
-    .then(stackOutputs => writeConfigFile(".env.template", ".env.local", stackOutputs))
+    .then(stackOutputs => writeConfigFile(".env.template", ".env", stackOutputs))
```

Comment:
Why this change? I used .env.local successfully with Vite elsewhere.

Reply:
Good point - updated